easybuild/tools/job/gc3pie.py

```diff
@@ -199,6 +200,10 @@ def make_job(self, script, name, env_vars=None, hours=None, cores=None):
 
         if cores:
             named_args['requested_cores'] = cores
+        elif build_option('job_cores'):
+            named_args['requested_cores'] = build_option('job_cores')
+        else:
+            # FIXME: ask GC3Pie to guess core count per node (see _get_ppn in job/pbs_python.py)
```

Comment:
> any ideas on this? does GC3Pie support this somehow?

I'm afraid this is not something that can be done with GC3Pie at the moment.  It could in principle be added, but would take quite some work as different batch-queueing systems have very different ideas over how to report the "max core count per node".